defaults/main.yml

```diff
@@ -1,11 +1,32 @@
-# Dont change this value, it's value is referenced
+
+# Tarsnap key contents
+# Should look like
+# TARSNAP_KEY: |
+#      # START OF TARSNAP KEY FILE
+#      contents
+#      # END OF TARSNAP KEY FILE
+TARSNAP_KEY: null
 TARSNAP_KEY_REMOTE_LOCATION: /root/tarsnap.key
-TARSNAP_KEY_LOCAL_TEMPLATE: null
 
-TARSNAPPER_CONF: null
+# You'll need to upload these scripts by yourself
+TARSNAP_BACKUP_PRE_SCRIPT: null
+TARSNAP_BACKUP_POST_SCRIPT: null
+
```

Comment:
I spend like hour yesterday to figure our a sensible way to have this role upload ``pre`` and ``post`` scripts without having these scripts in saved in this role (which is undesirable --- I agree). It's either my ansible-fu fails me or this is not doable. 

So user of this role needs to upload pre and post scripts himself